generators/client/templates/angular/src/main/webapp/app/app.constants.ts.ejs

```diff
@@ -24,3 +24,4 @@ export const VERSION = process.env.VERSION;
 export const DEBUG_INFO_ENABLED = !!process.env.DEBUG_INFO_ENABLED;
 export const SERVER_API_URL = process.env.SERVER_API_URL;
 export const BUILD_TIMESTAMP = process.env.BUILD_TIMESTAMP;
+// export const MY_DYNAMIC_VARIABLE = window.__env.myDynamicVariable;
```

Comment:
Instead of incorporating a change without any real integration, I think it would be good to specify external server url and debug settings in this `env.js` and remove those from the webpack build. If you generate a separate front end and backend, then, the capability to specify external url is very useful.  However, I think it as a breaking change, so, this PR should be deferred for `v7`.

Reply:
@vishal423 , Indeed it would be a breaking change. maybe you can add a PR with only that change once v7 is close